R/human_infection.R

```diff
@@ -468,7 +523,7 @@ severe_immunity <- function(age, acquired_immunity, maternal_immunity, parameter
   parameters$theta0 * (parameters$theta1 + (1 - parameters$theta1) / (
     1 + fv * (
       (acquired_immunity + maternal_immunity) / parameters$iv0) ** parameters$kv
-    )
+  )
```

Comment:
This whitespace actually helps to match brackets

```suggestion
    )
```